scipy/stats/_stats_py.py

```diff
@@ -10314,3 +10314,108 @@ def first_order(t):
     # finding a wrong root.
     res = root_scalar(first_order, x0=x0, x1=x1)
     return res.root
+
+
+def ChebyMeanTest_1samp(a, popmean, confidence_level=0.95,
+                        axis=0, nan_policy='propagate'):
```

Comment:
We have a decorator for the `axis` and `nan_policy`: `_axis_nan_policy_factory`. I would either suggest using it, or ensure all associated tests would pass on this function.

Also optional parameters should be marked as keyword only (`*`).